chapters/lit_review/angle_VR.tex

```diff
@@ -282,6 +282,59 @@ \subsubsection{Angular Biasing Using the Exponential Transform}
 evenly distributing the problems in problems with anisotropy.
 MCNP Weight Window Generator \textbf{cite MCNP manual here}
 
+
+The AVATAR method
+\cite{van_riper_generation_1995, van_riper_avatarautomatic_1997} used an
+approximation of the angular flux--without explicitly calculating it--to
+generate angle-dependent weight windows. It operated under the approximation
+that
+the angular flux was separable and symmetric about the average current vector.
+The angular flux was then calculated using
+a product of a deterministically-obtained
+scalar flux and an exponential function, derived from the
+maximum entropy distribution, that was a function of the scalar flux and the
+current. Space-, energy-, and angle-dependent weight windows for
+the Monte Carlo problem were then generated from the inverse of the angular
+flux. AVATAR improved the FOM for sample problems from 2 to 5 times, but did not
```

Comment:
All of these methods need some equations, not just qualitative descriptions. AVATAR and LIFT in particular as they're the most closely related to what you did. Including the math (a bit like the lit review that I did) is what gives the real context.